### ISSUE TITLE
Use with npm and leaflet 1.0.0-rc.3

### ISSUE THREAD

**Issue by jcoupey** [perliedman/leaflet-control-geocoder#143]
I'm using browserify and leaflet-control-geocoder which I installed locally with npm. In order to use other things that are only supported with the recent leaflet rc releases, I also forced the leaflet version:

```
npm install --save leaflet@1.0.0-rc.3
```

My problem is that the leaflet-control-geocoder `package.json` states:

```
"dependencies": {
  "leaflet": "~0.7.7"
}
```

This seems to force the leaflet version to 0.7.7 globally. When using `var L = require('leaflet')`, the `L` object in the DOM has its `version` member set to "0.7.7".

Is there a way to manage this use-case ?

**Comment by perliedman** [comment]
I think the best way to avoid this problem is to just drop the dependency on Leaflet, and let the plugin-user make sure that the right version of Leaflet is loaded.

As far as I can tell, the only downside of this i that it will not be possible to run multiple versions of Leaflet on the same page, but I think that's a pretty unusual use case and there would likely be other issues if you actually did this.

**Comment by jcoupey** [comment]
Thanks for your quick answer. This sounds fine to me, and I would only have to update the npm package to the new version with no leaflet dependency.

**Comment by perliedman** [comment]
Dropped the dependency and published 1.5.2, so this is done now.
